manifests/l/lin-ycv/EverythingCmdPal/0.1.0.0/lin-ycv.EverythingCmdPal.installer.yaml

```diff
@@ -10,7 +10,7 @@ MinimumOSVersion: 10.0.19041.0
 InstallerType: msix
 Dependencies:
   PackageDependencies:
-  - PackageIdentifier: Microsoft.PowerToys
+#  - PackageIdentifier: Microsoft.PowerToys

```

Comment:
PT installation from winget hangs, after removing this dependency, package installs properly
![Screenshot 2025-04-17 125651](https://github.com/user-attachments/assets/1acf2422-e46d-4c3d-9fa3-676f58046b06)

Opened an issue on PT repo: https://github.com/microsoft/PowerToys/issues/38920